mrmustard/lab_dev/circuit_components.py

```diff
@@ -159,6 +160,37 @@ def light_copy(self) -> CircuitComponent:
         instance.__dict__["_wires"] = self.wires.copy()
         return instance
 
+    def to_fock(self, shape: Optional[Union[int, Iterable[int]]] = None) -> CircuitComponent:
```

Comment:
should we call it `bargmann_to_fock`?

Reply:
I changed it into `to_fock_component`, since it also works on components with `Fock` representation, not only bargmann